### DISCUSSION TITLE
Вопрос по Cbz

### DISCUSSION THREAD

**Discussion by angry-master-yoda** [lirix360/ReadmangaGrabber#97] [Q&A]
Пока вообще не знаю, что это за формат... Посмотрим, что читалка скажет.

Вопрос.
Сейчас упаковка идёт по главам, внутри папка с номером главы. А весь том упаковать можно? И это будет потом читаться?

**Comment by lirix360** [comment]
[https://ru.wikipedia.org/wiki/Comic_Book_Archive](https://ru.wikipedia.org/wiki/Comic_Book_Archive)

Тома не видел и не пробовал, не знаю.

**Reply by angry-master-yoda** [reply]
Да, это нашёл уже :-).

С несколькими папками поэкспериментирую и отпишусь.

А ещё внутрь можно положить файл с информацией о манге. Но, как он должен выглядеть, пока что есть тайна, покрытая мраком.
